resqpy/rq_import/_add_surfaces.py

```diff
@@ -116,7 +116,6 @@ def _add_single_surface(model, surf_file, surface_file_format, surface_role, qua
     surface.create_xml(ext_uuid,
                        add_as_part = True,
                        add_relationships = True,
-                       crs_uuid = rqet.uuid_for_part_root(crs_root),
```

Comment:
Emma, Could you also please remove crs_root as an argument to this function _add_single_surface()

Then of course you will need to remove it in any calling code.  You can either make a breaking change by removing it from the argument list of the public equivalent function, or add a few lines there to convert to crs_uuid and produce a deprecation warning.